README.md

```diff
@@ -4,7 +4,7 @@ SPDX-FileCopyrightText: 2021 Philipp Basler, Margarete Mühlleitner and Jonas M
 SPDX-License-Identifier: GPL-3.0-or-later
 -->
 
-Program: BSMPT version 3.0.5
+Program: BSMPT version 3.0.6
```

Comment:
```suggestion
Program: BSMPT version 3.0.5
```

Reply:
I will merge this and adjusted the release version of #153 and will make this one release then.